gotchas/styled-jsx-in-ui-core.md

```diff
@@ -0,0 +1,9 @@
+# Usage of styled-jsx in our ui-core library
+
+With styled-jsx, for our components in ui-core:
+
+- We try to target the element (e.g. https://github.com/dhis2/ui-core/blob/master/src/Button/styles.js#L6)
+- Sometimes we cannot do that, e.g. if we have nested div elements, then we use cx('root', className) and add the .root to the outermost div
```

Comment:
I think this should be a sub-bullet-point of the first one, otherwise an unordered list doesn't really make sense here

Reply:
Done in e40624f323d6b3fbf36190408f1cc87d2b202a11